code/modules/unit_tests/quick_swap_sanity.dm

```diff
@@ -29,3 +29,21 @@
 	// Since the tank is a valid suit storage item, it should not be dropped
 	TEST_ASSERT(human.equip_to_appropriate_slot(coat, swap = TRUE), "Couldn't quick swap to coat")
 	TEST_ASSERT_EQUAL(human.s_store, tank, "Human dropped the oxygen tank, when it was a valid item to keep in suit storage")
+
+/// Tests that doUnEquip code is ran by checking vision correcting glasses
+/datum/unit_test/quick_swap_glasses/Run()
+	var/mob/living/carbon/human/human = allocate(/mob/living/carbon/human)
+
+	ADD_TRAIT(human, TRAIT_NEARSIGHT, TRAIT_GENERIC)
+
+	var/obj/item/clothing/glasses/regular/glasses = allocate(/obj/item/clothing/glasses/regular)
+	TEST_ASSERT(human.equip_to_slot_if_possible(glasses, ITEM_SLOT_EYES), "Couldn't equip glasses")
```

Comment:
Are you checking if human/equip_to_slot() is working here? or have I misunderstood.

Reply:
Pretty much, doing asserts like this is mostly free.